src/DataStructures/Tensor/EagerMath/DotProduct.hpp

```diff
@@ -75,6 +75,50 @@ Scalar<DataType> dot_product(
 }
```

Comment:
Please factor these types of commits into separate PRs so they aren't "snuck" into the code base. Thanks.

Reply:
Ok, here it is: https://github.com/sxs-collaboration/spectre/pull/1672

Reply:
Thanks!